thirdeye/thirdeye-frontend/app/pods/application/route.js

```diff
@@ -87,9 +87,9 @@ export default Route.extend(ApplicationRouteMixin, {
           const errorStatus = error.response ? error.response.status : 'Unknown';
           notifications.error(toastMsg, `${errorStatus} error detected`, toastOptions);
         }
-      } else {
-        throw new Error(error);
-      }
+        if (isDevEnv) {
+          throw new Error(error);
```

Comment:
I was thinking we can do something like `http://localhost:4200/#/self-serve/create-alert?isDevEnv=true`. This allows us at any route to override the `isDevEnv` to decide which mode to be in. This would not need inspector change.

and this design allows for other debugging flags

Reply:
i'll do this in another PR.